include/open62541/plugin/certificategroup.h

```diff
@@ -80,6 +80,18 @@ UA_EXPORT UA_StatusCode
 UA_CertificateUtils_ckeckKeyPair(const UA_ByteString *certificate,
                                  const UA_ByteString *privateKey);
 
+/* Determines if the Certificate Revocation List (CRL) provided belongs to the given certificate.
+ *
+ * @param certificate The byte string representing the certificate.
+ * @param crl The byte string representing the CRL.
+ * @return UA_STATUSCODE_GOOD if the CRL belongs to the certificate,
+ *         UA_STATUSCODE_BADNOMATCH if the CRL does not belong to the certificate,
+ *         UA_STATUSCODE_BADINTERNALERROR if an error occurs.
+ */
```

Comment:
You check whether certificate and crl have the same issuer.
That is not the same as matching against the (exact) certificate and can be misread.
Also make sure to mention that we do not check whether the certificate has been revoked by the crl.
That’s another way the intention of the method could be misinterpreted.